bokeh/core/_templates/doc_js.js

```diff
@@ -1,4 +1,17 @@
 var docs_json = {{ docs_json }};
 var render_items = {{ render_items }};
 
-Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+function do_embed() {
+    Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+}
+
+if (typeof Bokeh != 'undefined') {
+    do_embed()
+}
+else {
+    if (typeof window._bokeh_todos === 'undefined') {
+        console.log("Bokeh is not yet loaded, deferring call to embed_items")
+        window._bokeh_todos = []
+    }
+    window._bokeh_todos.push(do_embed)
+}
```

Comment:
@mattpap @havocp who can work on this today? I could but it would mean deferring other tasks.